lib/broccoli/ember-app.js

```diff
@@ -1078,7 +1078,8 @@ class EmberApp {
     });
 
     return new Funnel(mergedAddonStyles, {
-      destDir: this.name,
+      allowEmpty: true,
```

Comment:
should we regression test this empty tree?

Reply:
we should!